.github/workflows/ci.yml

```diff
@@ -10,12 +10,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
       - name: Install dependencies
         run: |
+          python -m pip install --upgrade pip
           sh scripts/install_requirements.sh
       - name: Lint
         run: |
-          echo "Linting is not implemented yet"
+          cpplint --filter=-legal,-build main.cpp
```

Comment:
So, we need to specify all files here? Is it possible to pass only the root folder?

Reply:
Btw, what do these filters do?

Reply:
Filter `legal` targets copyright issues, the second one `build` I decided to remove